erc20.go

```diff
@@ -78,17 +78,14 @@ func (p *Plugin) InitPlugin(metadata *persist.Metadata, bucket *bolt.Bucket, sto
 			Version: pluginDBVersion,
 			Header:  pluginDBHeader,
 		}
-	} else if metadata.Version != pluginDBVersion {
+	} else if metadata.Version != pluginDBVersion || metadata.Header != pluginDBHeader {
 		return persist.Metadata{}, errors.New("There is only 1 version of this plugin, version mismatch")
```

Comment:
Don't try to be too quick, now your error is no longer correct. Think first, solve second.